yt_dlp/downloader/external.py

```diff
@@ -513,3 +587,38 @@ def get_external_downloader(external_downloader):
     # Drop .exe extension on Windows
     bn = os.path.splitext(os.path.basename(external_downloader))[0]
     return _BY_NAME.get(bn)
+
+
+def parse_ffmpeg_time_string(time_string):
+    time = 0
+    reg1 = re.match(r"((?P<H>\d\d?):)?((?P<M>\d\d?):)?(?P<S>\d\d?)(\.(?P<f>\d{1,3}))?", time_string)
+    reg2 = re.match(r"\d+(?P<U>s|ms|us)", time_string)
```

Comment:
Unclear var name for reg1 and reg2
```suggestion
    hms_parsed = re.match(r"((?P<H>\d\d?):)?((?P<M>\d\d?):)?(?P<S>\d\d?)(\.(?P<f>\d{1,3}))?", time_string)
    smu_parsed = re.match(r"\d+(?P<U>s|ms|us)", time_string)
```